website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-enabling-continuous-integration-with-github.md

```diff
@@ -141,3 +141,11 @@ Select the **GitHub**, **GitLab**, or **AzureDevOps** tab and reselect your repo
 If you receive a schema-related error message referencing a *previous* PR, this is usually an indicator that you are not using a production job for your deferral and are instead using *self*.  If the prior PR has already been merged, the prior PR's schema may have been dropped by the time the Slim CI job for the current PR is kicked off.
 
 To fix this issue, select a production job run to defer to instead of self.
+
+## Limitations
+
+If your temporary PR schemas are not dropping after a merge/close of the PR, it could be one of the following:
```

Comment:
@runleonarun this in reference to https://docs.getdbt.com/docs/dbt-cloud/using-dbt-cloud/cloud-enabling-continuous-integration

Basically when dbt Cloud runs a Slim CI job (triggered by users opening a PR in their own repositories)... it creates temporary PR schemas in the users database that take the format of:

```
dbt_cloud_pr_<job_id>_<pr_number>
```

When the user decides to merge or close their PR in GitHub (or GitLab), dbt Cloud sends a `drop schema like dbt_cloud_pr_<job_id>_<pr_number>` statement over to their datawarehouse.

This effectively "cleans up" their database so as to not have lots and lots of `dbt_cloud_pr_<job_id>_<pr_number>` schemas lingering around. Occasionally, the users PR / temporary schemas in their database AREN'T dropped and I'm seeking to provide more information on the scenarios as to when this could occur. Hopefully didn't over-explain here :P